app/utils/session.js

```diff
@@ -1,7 +1,50 @@
 'use strict'
 
 // Local dependencies
+const { PAYMENT_EXTERNAL_ID } = require('@govuk-pay/pay-js-commons').logging.keys
+const logger = require('../utils/logger')(__filename)
+const { setLoggingField, getLoggingFields } = require('../utils/logging_fields_helper')
 const cookie = require('../utils/cookies')
 
-exports.createChargeIdSessionKey = chargeId => 'ch_' + chargeId
-exports.retrieve = (req, chargeId) => cookie.getSessionVariable(req, exports.createChargeIdSessionKey(chargeId))
+const createChargeIdSessionKey = function createChargeIdSessionKey( chargeId ){
```

Comment:
Inconsistent spacing and bracket conventions in this file, ideally these things would be caught by a lint. I believe this repo uses 

```suggestion
const createChargeIdSessionKey = function createChargeIdSessionKey(chargeId) {
```

Reply:
done